cypress/integration/experiences.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="Cypress" />
+
+import { ArticlePage } from '../page-objects/article-page'
+import { ArticleMenuPage } from '../page-objects/article-menu-page'
+
+const articlePage = new ArticlePage()
+const articleMenuPage = new ArticleMenuPage()
+
+describe('Article view', () => {
+  it('change article language', () => {
+    cy.setLocalStorage('has-onboard-before', true)
+    cy.visit('http://127.0.0.1:8080/#/article/en/Cat')
+    articlePage.selectOptionFromActionsMenu('languages')
+    cy.get('input').type('português')
+    cy.get('.description').should('have.text', 'Gato')
+    cy.downArrow().enter()
+    cy.clickDoneButton()
+    articlePage.title().should('have.text', 'Gato')
+  })
+
+  it('check footer', () => {
+    articlePage.selectOptionFromActionsMenu('sections')
```

Comment:
I just check on cypress, when you have a new `it()` scope, it clears all the local storage item.

Therefore, the following script is needed at the beginning of the scope

```js
cy.setLocalStorage('has-onboard-before', true)
cy.visit('http://127.0.0.1:8080/#/article/en/Cat')
```

Reply:
@jpita does this work for you?

Reply:
well, the `has-onboard-before` variable has nothing to do with the language.
shall we ignore this and merge the code?

Reply:
as per the meeting discussion, please ignore this and merge the code without experience.js.